Sources/SmileID/Classes/Helpers/LocalStorage.swift

```diff
@@ -133,6 +134,39 @@ public class LocalStorage {
         return try createSmileFile(to: jobId, name: "info.json", file: data)
     }
 
+    static func addSecurityInfo(
+        jobId: String,
+        files: [URL]
+    ) throws -> URL? {
+        do {
+            let timestamp = Date().toISO8601WithMilliseconds()
+            let mac = try SmileIDCryptoManager.shared.sign(
+                timestamp: timestamp,
+                files: files
+            )
+            let securityInfo = SecurityInfo(
+                timestamp: timestamp,
+                mac: mac
+            )
+            let securityInfoJson = try createSecurityInfoFile(
+                jobId: jobId,
+                securityInfo: securityInfo
+            )
+            return securityInfoJson
+        } catch {
+            print("Couldn't create security info. Continuing without it.")
+            return nil
+        }
+    }
+
+    private static func createSecurityInfoFile(
+        jobId: String,
+        securityInfo: SecurityInfo
+    ) throws -> URL {
+        let data = try jsonEncoder.encode(securityInfo)
+        return try createSmileFile(to: jobId, name: "security_info.json", file: data)
```

Comment:
@robin-smileid on this I think we should do in memory addition for this at the point of zipping files so that this file doesn't stay on the file system?

Reply:
Resolved in [4a0c549](https://github.com/smileidentity/ios/pull/286/commits/4a0c54974c5ef35858bc001f778c8587e88fc7b5)